src/retrace/retrace.py

```diff
@@ -2122,8 +2124,25 @@ def strip_extra_pages(self) -> None:
             log_warn("makedumpfile exited with %d" % child.returncode)
             if newvmcore.is_file():
                 newvmcore.unlink()
+            return 
+
+        #Calculate the % of storage space saved with the new makedumpfile core created. 
+        elif ((oldsize - newvmcore.stat().st_size) / int(oldsize) * 100 < CONFIG["VmcoreDumpSavePercent"]):
+            log_warn("makedumpfile did not strip enough pages (needed to save %{}). 
+                    Not saving new vmcore.".format(CONFIG["VmcoreDumpSavePercent"]))
+            format_number = "{:.2f}".format((int(oldsize) - int(newvmcore.stat().st_size)) / int(oldsize) * 100)
+            log_info("Percentage of space that would have been saved if we kept the stripped vmcore 
+                    is: %s" % format_number)
```

Comment:
I think this can be simplified slightly:
```suggestion
            saved_pct = (oldsize - newvmcore.stat().st_size) / oldsize * 100
            log_info(f"Keeping the stripped vmcore would have saved {saved_pct:.2f}% of space")
```